monkey/monkey_island/cc/ui/src/components/pages/ConfigurePage.js

```diff
@@ -72,12 +72,9 @@ class ConfigurePageComponent extends AuthComponent {
     };
   }
 
-  componentDidUpdate() {
-    if (!this.getSectionsOrder()?.includes(this.currentSection)) {
-      this.currentSection = this.getSectionsOrder()?.[0];
-      this.setState({selectedSection: this.currentSection});
-    }
-  }
+  // componentDidUpdate() {
+      // this.setState({selectedSection: this.currentSection});
+    // }
```

Comment:
I think we can remove this code. I can navigate between configuration tabs just fine